packages/calcite-components/src/components/slider/slider.e2e.ts

```diff
@@ -1126,4 +1127,265 @@ describe("calcite-slider", () => {
       });
     });
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-slider-handle-border-color": {
+          shadowSelector: `.${CSS.handle}`,
+          targetProp: "boxShadow",
+        },
+        "--calcite-slider-handle-border-color-hover": {
+          shadowSelector: `.${CSS.handle}`,
+          targetProp: "boxShadow",
+          state: "hover",
+        },
+        "--calcite-slider-handle-color": {
+          shadowSelector: `.${CSS.handle}`,
+          targetProp: "backgroundColor",
+        },
+        "--calcite-slider-handle-color-active": {
+          shadowSelector: `.${CSS.handle}`,
+          targetProp: "backgroundColor",
+          state: { press: { attribute: "class", value: CSS.handle } },
+        },
+        "--calcite-slider-handle-color-hover": {
+          shadowSelector: `.${CSS.handle}`,
+          targetProp: "backgroundColor",
+          state: "hover",
+        },
+        "--calcite-slider-track-color": {
+          shadowSelector: `.${CSS.track}`,
+          targetProp: "backgroundColor",
+        },
+      };
+      themed(html` <calcite-slider min="0" max="100" value="20" step="10"></calcite-slider> `, tokens);
+    });
+    describe("disabled", () => {
```

Comment:
Is it possible to merge some of these test together to speed testing up?

Reply:
For the `disabled` describe block, I had to separate it into 2 child describe blocks in order to test the same token being applied to 2 different classes, because you can't have duplicate json keys in the tokens setup object.